pwa/src/components/endpoints/endpointsTable.tsx

```diff
@@ -8,43 +8,41 @@ import { HeaderContext } from "../../context/headerContext";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faTrash, faEdit } from "@fortawesome/free-solid-svg-icons";
 import DeleteModal from "../deleteModal/DeleteModal";
-import LoadingOverlay from "../loadingOverlay/loadingOverlay";
```

Comment:
You have deleted the loadingOverlay wich means that when you delete an item you will be able to edit that item you are about to delete.

Reply:
Nice catch. Fixed.